x-pack/plugins/triggers_actions_ui/public/common/truncated_text.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+// eslint-disable-next-line @kbn/eslint/module_migration
+import styled from 'styled-components';
+
+const LINE_CLAMP = 2;
+
+const Text = styled.span`
```

Comment:
I was under the impression that we want to avoid using styled components and use the `styles` prop directly. Correct me if I'm wrong.

Reply:
@XavierM do you know if this is true or not?

Reply:
My bad here, you are right, we also used different technique like

```ts
import { css } from '@emotion/react';

export const styles = {
  buttonContainer: css`
    display: inline-block;
    position: relative;
  `,
};
...
import { styles } from './field_browser.styles';
<div css={styles.buttonContainer}></div>
```

Reply:
Fixed here: 267a82876bec6a453b040e8cdaf55be70e92cf2c